### PR TITLE
[plugin.video.lbry] v0.1.5+leia.1

### PR DESCRIPTION
**Pull request by accumulator**
### Description

- fix error in log statement
- fix exception thrown if channel contains a live stream
- add menu option to list recent videos from all subscribed channels
- add support for reading and posting comments

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Review by romanvm — APPROVED** [review]
LGTM

**Comment by basrieter** [comment]
@accumulator  Please make the Add-on Checker happy and fix the error.

**Review by basrieter — CHANGES_REQUESTED** [review]
Please fix the add-on checker issue.

File: plugin.video.lbry/resources/skins/Default/skin.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (enen92) [inline plugin.video.lbry/resources/skins/Default/skin.xml:1]: this file is no longer used by addons please remove from the submission

**Review by enen92 — CHANGES_REQUESTED** [review]
see comments

**Pull request closed without merging**